docs/tables_and_indices.md

```diff
@@ -264,6 +264,20 @@ storage of rows; if there is no good way to partition the rows uniformly and
 range requests are not needed, then perhaps
 `partitionStrategy(PartitionStrategy.HASH)` is a better idea for your table.
 
+{{site.data.alerts.important}}
+The first defined rowComponent of any table is used by the partitioner to distribute
+data across the cluster. To avoid hot-spotting, the type of the first row component
```

Comment:
nit: it's not just the first row component, but it is the most significant component